README.md

```diff
@@ -56,10 +56,23 @@ To verify that Sonobuoy has completed successfully, check the logs:
 ```
 kubectl logs -f sonobuoy --namespace=heptio-sonobuoy
 ```
-If you see the log line `no-exit was specified, sonobuoy is now blocking`, the Sonobuoy pod is done running.
+If you see the log line `no-exit was specified, sonobuoy is now blocking`, the Sonobuoy pod has completed its data collection.
 
-*Note*: If you see the error `plugin <name> does not exist`, delete
-the `sonobuoy` pod via `kubectl delete pod sonobuoy --namespace heptio-sonobuoy` and rerun the `kubectl apply` command above. This error occurs when Sonobuoy's ConfigMap creations are not completed before its pod creation. While the YAML files in `examples/quickstart` use numeric prefixes ("00-", "10-", etc.) to specify the order of resource creation, `kubectl apply` is asynchronous and can result in dependency issues.
+> *Notes*:
+>
+> * **The Sonobuoy pod in this example continues to run after it finishes data collection**, due to the `--no-exit` flag in its YAML manifest. This allows you to easily grab the results tarball.
```

Comment:
I'd like to add the preferred to use persistent volume clause here if we're making a separate note about no-exit.